spec/models/requests/hold_item_spec.rb

```diff
@@ -85,6 +87,8 @@
         expect(hold_request.submitted.size).to eq(1)
         expect(hold_request.errors.size).to eq(0)
         expect(hold_request.submitted.first[:payload]).to include("<last-interest-date>#{(todays_date + 7).strftime('%Y%m%d')}</last-interest-date>")
+        expect(hold_request.submitted.first[:payload]).not_to include("<pickup-location>489</pickup-location>")
```

Comment:
@christinach my apologies.  I meant just remove the pickup in the test below.  Could you put it back here?  Sorry!  I thought it would be good for one test to test the case where it was set, and the other to test the case where it used the default.

Reply:
Yes @carolyncole , I put it back. thank you!